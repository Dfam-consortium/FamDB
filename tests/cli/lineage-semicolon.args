lineage
--format
semicolon
-a
5
