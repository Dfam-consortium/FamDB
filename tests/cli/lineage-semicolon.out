7(2): root;Order;Other Genus;Other Species [1]
